csharp/compatibility_tests/v3.0.0/test.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+function run_test() {
+  # Generate test proto files.
+  ./protoc_1 -Iprotos/src -I../../../src/ --csharp_out=src/Google.Protobuf.Test \
+    --csharp_opt=base_namespace=Google.Protobuf \
+    protos/src/google/protobuf/map_unittest_proto3.proto \
+    protos/src/google/protobuf/unittest_import_proto3.proto \
+    protos/src/google/protobuf/unittest_import_public_proto3.proto \
+    protos/src/google/protobuf/unittest_well_known_types.proto
+
+  ./protoc_1 -Iprotos/csharp --csharp_out=src/Google.Protobuf.Test \
+    --csharp_opt=base_namespace=UnitTest.Issues \
+    protos/csharp/protos/unittest_issues.proto
+
+  ./protoc_2 -Iprotos/src --csharp_out=src/Google.Protobuf.Test \
```

Comment:
Okay - it's not entirely clear what the motivation is for the different sets, but I'm fine with it. (And the PR in general.)